src/Style/Colors.test.ts

```diff
@@ -0,0 +1,57 @@
+import { PrimaryColor, SecondaryColor, Greyscale } from './Colors';
+
+describe('Colors', () => {
+  test('PrimaryColor', () => {
+    expect(PrimaryColor).toEqual({
+      glintsyellow: '#FFF240',
+      glintsred: '#EC272B',
+      glintsblue: '#0BAEEC',
+    });
+  });
+
+  test('SecondaryColor', () => {
+    expect(SecondaryColor).toEqual({
+      actionblue: '#017EB7',
+      green: '#93BD49',
+      darkgreen: '#008464',
+      magenta: '#FF00E5',
+      violet: '#9013FE',
+      orange: '#F5A623',
+
+      // deprecated
+      ivory: '#FFFAEA',
+      lighterblack: '#AAAAAA',
+      lightergrey: '#EEEEEE',
+      blue: '#0869C2',
+      darkerblue: '#027EB7',
+      lightgreen: '#EEF6F4',
+      lightorange: '#FFF6E8',
+      lightred: '#FFF2F3',
+      lightblue: '#D6F4FF',
+      darkblue: '#004b68',
+
+      // renamed
+      purplepizza: '#FF00E5',
+      bluegem: '#9013FE',
+
+      // moved to Greyscale
+      white: '#FFFFFF',
+      whitesmoke: '#F3F3F3',
+      black: '#000000',
+      lightblack: '#777777',
+      grey: '#777777',
+      darkgrey: '#777777',
+      lightgrey: '#C6C6C6',
+    });
+  });
+
+  test('Greyscale', () => {
+    expect(Greyscale).toEqual({
+      black: '#000000',
+      grey: '#777777',
+      lightgrey: '#C6C6C6',
+      softgrey: '#F3F3F3',
+      white: '#FFFFFF',
+    });
+  });
+});
```

Comment:
Maybe it would make more sense to go with a snapshot test here? I think you could just do something like `expect(Greyscale).toMatchSnapshot()`. That would have the advantage that you wouldn't have to manually adjust colors when they change, just have to update the snapshot. And you'd still accomplish the goal of having failing tests when the Colors objects change.

Reply:
thank u! let me try it, didn't know we can do this `expect(Greyscale).toMatchSnapshot()`

Reply:
![Screen Shot 2020-04-23 at 11 09 22 AM](https://user-images.githubusercontent.com/11376798/80054850-ed1af200-8552-11ea-94df-5072691fef54.png)

@rosslo do you know why this color snapshot failed, does it run test against complied js file?

Reply:
oh yes it runs the `lib/Style/Colors.test.js`

Reply:
@westwood846 I will leave the color test as is, we need to change our test setting.

Reply:
Haha, I think we should not include test files in packages (lib/ es).